### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
In the code example of topic "exiting loops with break". The curly brackets were missing in an "else if" conditionals. I found it while learning about loops.

#### Motivation
I am new to programming, and found this error while learning. 

#### This PR:
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/javascript/building_blocks/looping_code/index.md [hunk 1]
```diff
@@ -302,8 +302,9 @@ btn.addEventListener('click', function() {
     if (splitContact[0].toLowerCase() === searchName) {
       para.textContent = splitContact[0] + '\'s number is ' + splitContact[1] + '.';
       break;
-    } else if (i === contacts.length-1)
+    } else if (i === contacts.length-1) {
       para.textContent = 'Contact not found.';
+      }
```
Comment (wbamberg) [inline files/en-us/learn/javascript/building_blocks/looping_code/index.md:307]:
```suggestion
    }
```

**Review by wbamberg — COMMENTED** [review]
Thanks for filing this! It's actually correct as it stands, I think: you don't need the brackets for a 1-line block.

I do like your version better, but this change will put the code out of sync with the version at https://github.com/mdn/learning-area/blob/master/javascript/building-blocks/loops/contact-search.html . If you would like to file a PR to update that version too, I'll merge this.

(@Rumyra , nice example of the pain of duplicating example sources :).

File: files/en-us/learn/javascript/building_blocks/looping_code/index.md (hunk 1 above)
Comment (pradeeps4ini) [reply files/en-us/learn/javascript/building_blocks/looping_code/index.md:307]: Hi! Thanks for reviewing the PR.  In the code of live example; curly brackets are already present in the "else-if" conditional. There's is no need to file a PR to update it.

Comment (wbamberg) [reply files/en-us/learn/javascript/building_blocks/looping_code/index.md:307]: Oh! You're right. Sorry @pradeeps4ini !

**Review by wbamberg — APPROVED** [review]
👍 thank you @pradeeps4ini !

**Comment by wbamberg** [comment]
> (Rumyra , nice example of the pain of duplicating example sources :).

apparently not!

**Pull request merged**